### PR TITLE
Improve error message unsupported attribute for bridges

### PR DESCRIPTION
**Pull request by blegat**
<s>Closes</s> <s>`https://github.com/jump-dev/JuMP.jl/issues/3550`</s>

### CODE REVIEW

**Comment by odow** [comment]
I can fix the rest of this

**Comment by odow** [comment]
This isn't what I had in mind for JuMP. We need a better error message at the JuMP level.

**Comment by blegat** [comment]
> This isn't what I had in mind for JuMP. We need a better error message at the JuMP level.

Then we can keep the issue open at the JuMP level but I think it's good to improve at the MOI level at least

File: src/Bridges/bridge.jl [hunk 1]
```diff
@@ -134,18 +145,8 @@ function MOI.get(
...
-            "the attribute `$attr`. If you encountered this error " *
-            "unexpectedly, it probably means your model has been " *
-            "reformulated using the bridge, and you are attempting to query " *
-            "an attribute that we haven't implemented yet for this bridge. " *
-            "Please open an issue at https://github.com/jump-dev/MathOptInterface.jl/issues/new " *
-            "and provide a reproducible example explaining what you were " *
-            "trying to do.",
-        ),
-    )
+    message = _attribute_error_message(attr, typepof(bridge), "accessing")
```
Comment (odow) [inline src/Bridges/bridge.jl:148]:
```suggestion
    message = _attribute_error_message(attr, typeof(bridge), "accessing")
```

**Review by odow — APPROVED** [review]

**Pull request merged**